app/src/main/cpp/vraze_app.cc

```diff
@@ -258,3 +263,33 @@ void VRazeApp::SetUpViewPortAndScissor(const gvr::Sizei &framebuf_size,
   // TODO: Discover how to make this work
   // renderer_->ScissorOn({left, bottom}, {width, height});
 }
+
+void VRazeApp::SendMessage(float x,float y, float dir0,float dir1){
+  // int status=this->vm->GetEnv((void**)&env,JNI_VERSION_1_6);
+  /*if(status<0)
+      status=this->vm->AttachCurrentThread(&env,vm);
+      if(status<0)
+          return;*/
+
+  vm->AttachCurrentThread(&env, 0);
+
+  //jstring mensaje = this->env->NewStringUTF("Este mensaje viene desde c++ :)");
+  //jfloatArray x = this->env->NewFloatArray(2);
+  // First get the class that contains the method you need to call
+  jclass clazz;// = env->FindClass("pe/edu/ucsp/vraze/MainActivity");
+
+  clazz = (jclass) env->NewGlobalRef(env->FindClass("pe/edu/ucsp/vraze/MainActivity"));
+  if(!clazz){
+    LOGD("HORROR - CANNOT FIND CLASS");
+  }
+
+  // Get the method that you want to call "(Ljava/lang/String;)V"
+  jmethodID metodo = env->GetStaticMethodID(clazz, "messageMe", "(FFFF)V");
```

Comment:
you can get this ID on the constructor, store it in a class member and then just call `CallStaticVoidMethod` here